cloud/src/meta-service/meta_service_resource.cpp

```diff
@@ -1085,11 +1119,11 @@ void MetaServiceImpl::alter_storage_vault(google::protobuf::RpcController* contr
     }
     case AlterObjStoreInfoRequest::ALTER_S3_VAULT: {
         alter_s3_storage_vault(instance, std::move(txn), request->vault(), code, msg);
-        return;
+        break;
```

Comment:
ret = alter_s3_storage_vault()
if ret != 0 should not break => nothing should be changed to isntancePB